src/test/java/utility/DatabaseUtility.java

```diff
@@ -1,58 +1,32 @@
 package utility;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 import java.sql.*;
 
 public class DatabaseUtility {
-    public Statement statement;
-    private static String jdbcUrl = ConfigurationReader.getProperty("jdbcUrl");
-    private static String username = ConfigurationReader.getProperty("dbusername");
-    private static String password = ConfigurationReader.getProperty("dbpassword");
-    private static Connection connection;
-    public void connect() {
-        try {
-            connection = DriverManager.getConnection(jdbcUrl, username, password);
-        } catch (SQLException e) {
-            e.printStackTrace();
-            throw new RuntimeException("Failed to connect to the database.");
-        }
-    }
+    private static final Logger logger = LogManager.getLogger(DatabaseUtility.class);
```

Comment:
Bunun yerine `@Slf4j` anotasyonunu kullanabiliriz

Reply:
Mesajı raporda da görmek icin Reporter.log() kullanmamız daha iyi olabilir sanırım. @Slf4j loglamasi testng-result raporuna yansıtır mi sonucu?

Reply:
Hayır o kısımda herhangi bir yansıtma yapmıyor fakat hem loglara basıp, hem de testng-result'a basarsak uygulama localde çalışırken daha rahat olur sizin için

Reply:
Tamam o sekilde düzenleyip yollayacağım